packages/kolibri-common/components/MissingResourceAlert.vue

```diff
@@ -7,16 +7,21 @@
       type="warning"
       :style="{ marginBottom: 0, marginTop: '8px' }"
     >
-      <span>
-        {{ coreString(
-          multiple ? 'someResourcesMissingOrNotSupported' :
-          'resourceNotFoundOnDevice'
-        ) }}
+      <span v-if="$slots.syncAlert">
+        <slot name="syncAlert"></slot>
+      </span>
+      <span v-else-if="syncCouldNotComplete">
```

Comment:
Not sure why this is needed? It is defaulting to `false`, so the default behaviour is still impacted here.

I think this can just be a `v-else` and the new prop can be removed.